### PR TITLE
test(clickhouse-logger): to show that different endpoints will be chosen randomly

### PR DESCRIPTION
**Pull request by xiaoxuanzi**
### Description

Add test for clickhouse-logger to show that different endpoints will be chosen randomly.

As mentioned in https://github.com/apache/apisix/pull/8604#discussion_r1089663859

Fixes # (issue)

### Checklist

- [x] I have explained the need for this PR and the problem it solves
- [ ] I have explained the changes or the new features added to this PR
- [ ] I have added tests corresponding to this change
- [ ] I have updated the documentation to reflect this change
- [ ] I have verified that this change is backward compatible (If not, please discuss on the [APISIX mailing list](https://github.com/apache/apisix/tree/master#community) first)

### CODE REVIEW

**Review by soulbird — DISMISSED** [review]

File: t/plugin/clickhouse-logger.t [hunk 1]
```diff
@@ -214,7 +214,38 @@ passed
 
 
 
-=== TEST 6: access local server
```
Comment (spacewander) [inline t/plugin/clickhouse-logger.t:217]: We prefer to add new test at the bottom to avoid affecting existing tests.

Comment (xiaoxuanzi) [reply t/plugin/clickhouse-logger.t]: Done

**Review by spacewander — DISMISSED** [review]

**Comment by moonming** [comment]
@shreemaan-abhishek please help to resolve conflicts then we can merge this PR

**Comment by shreemaan-abhishek** [comment]
@xiaoxuanzi please resolve the conflicts.

**Review by monkeyDluffy6017 — APPROVED** [review]

**Review by monkeyDluffy6017 — APPROVED** [review]

**Review by shreemaan-abhishek — DISMISSED** [review]

**Review by monkeyDluffy6017 — CHANGES_REQUESTED** [review]
It looks like that the ci failed, could you fix the ci? @shreemaan-abhishek

**Review by monkeyDluffy6017 — APPROVED** [review]

**Review by AlinsRan — APPROVED** [review]

**Pull request merged**